apps/judicial-system/backend/src/app/modules/case/filters/case.filters.spec.ts

```diff
@@ -453,51 +469,37 @@ describe('isCaseBlockedFromUser', () => {
     },
   )
 
-  each`
-    type
-    ${CaseType.SEARCH_WARRANT}
-    ${CaseType.BANKING_SECRECY_WAIVER}
-    ${CaseType.PHONE_TAPPING}
-    ${CaseType.TELECOMMUNICATIONS}
-    ${CaseType.TRACKING_EQUIPMENT}
-    ${CaseType.PSYCHIATRIC_EXAMINATION}
-    ${CaseType.SOUND_RECORDING_EQUIPMENT}
-    ${CaseType.AUTOPSY}
-    ${CaseType.BODY_SEARCH}
-    ${CaseType.INTERNET_USAGE}
-    ${CaseType.RESTRAINING_ORDER}
-    ${CaseType.EXPULSION_FROM_HOME}
-    ${CaseType.ELECTRONIC_DATA_DISCOVERY_INVESTIGATION}
-    ${CaseType.VIDEO_RECORDING_EQUIPMENT}
-    ${CaseType.OTHER}
-  `.describe('given an accepted $type case', ({ type }) => {
-    each`
+  each([...indictmentCases, ...investigationCases]).describe(
```

Comment:
This does not seem to work. When logging out the type parameter below, I get undefined.